From 394427a0dcfcb75db220c5b6a9622c5cf013a20f Mon Sep 17 00:00:00 2001
From: Aichun Shi <aichun.shi@intel.com>
Date: Mon, 14 Nov 2022 10:35:25 +0800
Subject: [PATCH 0457/1646] [Backport] Enable Intel Emmitsburg pinctrl for
 default config

Intel inclusion
category: feature
bugzilla: https://gitee.com/openeuler/intel-kernel/issues/I610P3

Reference: https://gitee.com/openeuler/kernel/commit/a32d3483e810e9025c7bcee2e6dce645e3f42fb1

--------------------------------------------

Enable necessary kernel configs for Intel Emmitsburg pinctrl in
openeuler_defconfig.

Signed-off-by: Aichun Shi <aichun.shi@intel.com>
Signed-off-by: Jialin Zhang <zhangjialin11@huawei.com>
---
 arch/x86/configs/openeuler_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 945cfded56ae..b53979933bf8 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -3762,7 +3762,7 @@ CONFIG_PINCTRL_BROXTON=m
 CONFIG_PINCTRL_CANNONLAKE=m
 CONFIG_PINCTRL_CEDARFORK=m
 CONFIG_PINCTRL_DENVERTON=m
-# CONFIG_PINCTRL_EMMITSBURG is not set
+CONFIG_PINCTRL_EMMITSBURG=m
 CONFIG_PINCTRL_GEMINILAKE=m
 CONFIG_PINCTRL_ICELAKE=m
 # CONFIG_PINCTRL_JASPERLAKE is not set
-- 
2.17.1

